### PR TITLE
Add some informations for google photos

### PR DESCRIPTION
**Pull request by Tontonjo**
On my Nothing phone, this app is used to quick edit screens captures which is getting broken if this package is removed

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -17102,7 +17102,7 @@
   {
     "id": "com.google.android.apps.photos",
     "list": "Google",
-    "description": "Google Photos (https://play.google.com/store/apps/details?id=com.google.android.apps.photos)",
+    "description": "Google Photos (https://play.google.com/store/apps/details?id=com.google.android.apps.photos), May break screen capture quick edit if removed",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json]:
```suggestion
    "description": "Google Photos (https://play.google.com/store/apps/details?id=com.google.android.apps.photos)\nMay break screen capture quick edit if removed.",
```

Also, maybe you can change the removal to `advanced`?

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Comment by AnonymousWP** [comment]
Can you still change the removal if required?

**Comment by Tontonjo** [comment]
> 
Sorry what do you mean?

**Comment by AnonymousWP** [comment]
> > 
> 
> Sorry what do you mean?

See https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen

**Comment by Tontonjo** [comment]
> > > 
> > 
> > 
> > Sorry what do you mean?
> 
> See https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen

I've read it but i still cant understand "cant you still change the removal" in the given context.
If you're talking about the recommendation which your link points to, i'd say it may be rated under "Expert", but i guess it depends on the app used by default by the phone in order to quick edit.

If you're asking if the package can easilly be reactivated: the answer is yes.

**Comment by AnonymousWP** [comment]
> > > > 
> > > 
> > > 
> > > Sorry what do you mean?
> > 
> > 
> > See https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-are-the-recommendations-chosen
> 
> I've read it but i still cant understand "cant you still change the removal" in the given context. If you're talking about the recommendation which your link points to, i'd say it may be rated under "Expert", but i guess it depends on the app used by default by the phone in order to quick edit.
> 
> If you're asking if the package can easilly be reactivated: the answer is yes.

With `removal` (which is the name of the key in the JSON file) to indicate the recommendation, yeah. Maybe you can change it to `expert`, but mention it in the description that it depends whether you set the app as default or not.

**Comment by AnonymousWP** [comment]
@Tontonjo Why did you do this?

**Comment by Tontonjo** [comment]
> @Tontonjo Why did you do this?

Oupsy, forgot about that, i was doing some cleanings in my repo's....

**Pull request closed without merging**